.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: Build Executables
+on:
+  release:
+    types: [published]
```

Comment:
[temporary] lets try doing a publish from branch, to test e2e?

Reply:
i have tested on branch already. https://github.com/percy/cli/actions/runs/5998822360